ros_cross_compile/ros_cross_compile.py

```diff
@@ -126,32 +111,39 @@ def parse_args(args: List[str]) -> argparse.Namespace:
     return parser.parse_args(args)
 
 
-def main():
-    """Start the cross-compilation workflow."""
-    args = parse_args(sys.argv[1:])
+def cross_compile_pipeline(
```

Comment:
Any reason to remove docstrings?

Reply:
Just didn't seem to be any more informative than the function name